src/com/apeshko/javacore/task_01/Application.java

```diff
@@ -0,0 +1,244 @@
+                            .withCapacity(2)
+                            .withType(Helicopter.Type.HEALTH)
+                            .withHeight(2000)
+                            .withWidth(300)
+                            .withLength(3567.9)
+                            .withWeight(1400)
+                            .withDistance(1000)
+                            .build(),
+                    new Helicopter.Builder()
+                            .withName("Friend")
+                            .withCapacity(4)
+                            .withType(Helicopter.Type.HEALTH)
+                            .withHeight(2200)
+                            .withWidth(400)
+                            .withLength(2567.9)
+                            .withWeight(1400)
+                            .withDistance(2000)
+                            .build()
+                )
+            ),
+            new AirCompany(
+                "Military Services",
+                Arrays.asList(
+                    new Helicopter.Builder()
+                            .withName("Warrior")
+                            .withCapacity(10)
+                            .withType(Helicopter.Type.MILITARY)
+                            .withHeight(3000)
+                            .withWidth(500)
+                            .withLength(4567.9)
+                            .withWeight(2400)
+                            .withDistance(3000)
+                            .build(),
+                    new Helicopter.Builder()
+                            .withName("Axe")
+                            .withCapacity(6)
+                            .withType(Helicopter.Type.MILITARY)
+                            .withHeight(3200)
+                            .withWidth(600)
+                            .withLength(3567.9)
+                            .withWeight(2600)
+                            .withDistance(3000)
+                            .build(),
+                    new Helicopter.Builder()
+                            .withName("Helper")
+                            .withCapacity(8)
+                            .withType(Helicopter.Type.MILITARY)
+                            .withHeight(3200)
+                            .withWidth(600)
+                            .withLength(3567.9)
+                            .withWeight(2800)
+                            .withDistance(2000)
+                            .build(),
+                    new Quadcopter.Builder()
+                            .withName("Mass Attack")
+                            .withCapacity(0)
+                            .withType(Quadcopter.Type.MILITARY)
+                            .withHeight(100)
+                            .withWidth(100)
+                            .withLength(100.9)
+                            .withWeight(20)
+                            .withDistance(500)
+                            .build()
+                )
+            )
+    );
```

Comment:
try to encapsulate initialization logic, f.i.
interface InitializationService {
  void init();
}

class MockInitializationService {
  ... implement this method
}

next you can easily switch initialization logic, get data from file, database, etc, just add additional interface implementation